daemons/gptp/common/ptp_message.cpp

```diff
@@ -873,7 +875,7 @@ void PTPMessageFollowUp::sendPort(IEEE1588Port * port,
 	XPTPD_INFO("Follow-up Dump:\n");
 #ifdef DEBUG
 	for (int i = 0; i < messageLength; ++i) {
-		XPTPD_PRINTF("%d:%02x ", i, (unsigned char)buf[i]);
+		fprintf(stderr, "%d:%02x ", i, (unsigned char)buf_t[i]);
```

Comment:
Is there a reason to remove the macro implementation? With a macro others can override implementation more easily.